gerrit-entrypoint.sh

```diff
@@ -83,13 +83,18 @@ EOF
 done
 
   #Customize gerrit.config
+  [ -z "${MAKE_FIRST_USER_ADMIN}" ] || set_gerrit_config capability.makeFirstUserAdmin "${MAKE_FIRST_USER_ADMIN}"
+  [ -z "${ENABLE_SIGNED_PUSH}" ] || set_gerrit_config receive.enableSignedPush "${ENABLE_SIGNED_PUSH}"
+  [ -z "${DOWNLOAD_SCHEME}" ] || set_gerrit_config download.scheme "${DOWNLOAD_SCHEME}"
 
   #Section gerrit
   [ -z "${WEBURL}" ] || set_gerrit_config gerrit.canonicalWebUrl "${WEBURL}"
   [ -z "${GITHTTPURL}" ] || set_gerrit_config gerrit.gitHttpUrl "${GITHTTPURL}"
 
   #Section sshd
-  [ -z "${LISTEN_ADDR}" ] || set_gerrit_config sshd.listenAddress "${LISTEN_ADDR}"
+  [ -z "${SSHD_LISTEN_ADDR}" ] || set_gerrit_config sshd.listenAddress "${SSHD_LISTEN_ADDR}"
```

Comment:
I agree that SSHD_LISTEN_ADDR is better than LISTEN_ADDR. However I'd like keep LISTEN_ADDR works in order to not break those exist containers to upgrade to the new image.